rhoas-cli/README.adoc

```diff
@@ -72,68 +72,44 @@ NOTE: The `rhoas` link:https://github.com/redhat-developer/app-services-cli/rele
 [discrete,id="installing-rhoas-cli-linux_{context}"]
 === Installing the rhoas CLI on Linux
 
-If you are using Linux, you can install `rhoas` by either using the installation script, or the RPM.
-The installation script installs the latest version of `rhoas` by default.
+.Procedure
 
-[discrete,id="installation-script_{context}"]
-==== Installation script
+. Navigate to the https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview[Red Hat OpenShift Streams for Apache Kafka] page on the Red Hat Developer Portal.
 
-.Procedure
+. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/Specifics*] and navigate to the bottom of the page to the *All Downloads* section.
 
-. Review the https://github.com/redhat-developer/app-services-cli/blob/main/scripts/install.sh[`install.sh` installation script^].
+. Click the *Download* button from the Linux row in the *RHOAS CLI* downloads table to download the `rhoas` zip file.
 
-. Run the `install.sh` script:
+. Unzip the file to extract the `rhoas` binary: 
 +
 --
 [source,shell]
 ----
-$ curl -o- https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/scripts/install.sh | bash
+$ unzip rhoas-${version}-linux.zip
```

Comment:
Rather than including the command prompt and command output, it might be better to just show the command itself. That way if someone uses the copy button, they only get the actual executable command. So, something like this:

----
unzip rhoas-${version}-linux.zip
----

Reply:
Yeah, makes sense.